### ISSUE TITLE
in progress project?

### ISSUE THREAD

**Issue by jxlijunhao** [SUSA-org/polygon-rnn#1]
Hi, is the project in progress or terminated?

After reading the polygon-rnn paper carefully, I think there left much work to do
1) the first vertex point detection CNN framework
2) training dataset,  convert the polygon point to the target boundary map, one-hot encoding etc.
3) loss function 
4) skip connection convLSTM

I'm reimplementing this paper and hope to share
